docs/source/models/supported_models.rst

```diff
@@ -232,6 +232,11 @@ Multimodal Language Models
     - Video
     - :code:`llava-hf/LLaVA-NeXT-Video-7B-hf`, etc. (see note)
     -
+  * - :code:`LlavaOnevisionForConditionalGeneration`
+    - LLaVA-Onevision
+    - Image\ :sup:`+` / Video\ :sup:`+`
+    - :code:`llava-hf/llava-onevision-qwen2-7b-ov-hf`, :code:`llava-hf/llava-onevision-qwen2-0.5b-ov-hf`, etc.
```

Comment:
I assume that the transformers version used for LLaVA-NeXT-Video and Qwen2-VL also work for this model? Can you update this row and the notes accordingly?

Reply:
OK. I have checked this transformers version, it works for this model.